<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application</class>
 <widget class="QMainWindow" name="application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>721</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="title">
     <string>choose a method </string>
    </property>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 127);</string>
     </property>
     <property name="text">
      <string>clean</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="choose_photo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 127);</string>
        </property>
        <property name="text">
         <string>Choose photo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="make_guess">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 127);</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>160</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="capture_photo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 127);</string>
        </property>
        <property name="text">
         <string>capture photo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_histogram">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 127);</string>
        </property>
        <property name="text">
         <string>show Histogram</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>44</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="hog">
        <property name="text">
         <string>Hog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lbp">
        <property name="text">
         <string>LBP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>721</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="photo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #CCCCCC; /* Light gray background */
color: #333333; /* Dark gray text */</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="camera">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #CCCCCC; /* Light gray background */
color: #333333; /* Dark gray text */</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>751</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Histogram</string>
    </property>
    <widget class="QLabel" name="histogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
